src/d3a/models/const.py

```diff
@@ -57,16 +59,16 @@ class StorageSettings:
         RATE_DECREASE_OPTION = 1
         # Energy sell break-even point, storage never sells for less than this value.
         # Unit is ct/kWh.
-        BREAK_EVEN_SELL = 25
+        BREAK_EVEN_SELL = 25.1
```

Comment:
Can you please elaborate the reasons for changing the default values here?
Also: As in the storage strategy, the BREAK_EVEN_SELL was renamed to final_selling_rate and BREAK_EVEN_BUY to final_buying_rate, how about renaming it here as well!?